static/js/ui_init.js

```diff
@@ -137,6 +163,9 @@ function message_hover(message_row) {
         msg_id: id,
     };
     message_row.find(".edit_content").html(render_edit_content_button(args));
+
+    const reaction_emoji = message_row.find(".message_reaction img.emoji");
+    animate(reaction_emoji);
```

Comment:
This name doesn't seem right; it should be something like `activate_emoji_if_animated`, since it doesn't do anything for non-animated emoji.

Reply:
How about `play_animation`? or `play_animation_if_animated`? I'd probably make sense to also change `stop_animation` to be `stop_animation_if_animated`.
An alternative for the above would be `replace_animation_with_still` and `replace_still_with_animation` but I'm not sure if that suffers from the same problem as above.

Have not made any changes for this in the last push.